integration_tests/src/main/python/delta_lake_write_test.py

```diff
@@ -54,7 +59,15 @@ def del_keys(key_list, c_val, g_val):
 
 def fixup_operation_metrics(opm):
     """Update the specified operationMetrics node to facilitate log comparisons"""
-    for k in "executionTimeMs", "numOutputBytes", "rewriteTimeMs", "scanTimeMs":
+    # note that we remove many byte metrics because number of bytes can vary
+    # between CPU and GPU.
+    # we remove deletion vector metrics because we do not support those yet
+    # see https://github.com/NVIDIA/spark-rapids/issues/8554
```

Comment:
Seems like there's two phases to supporting deletion vectors.  There's full support which is tracked by #8554, but before that is completed we should be reporting the metrics (always 0) in the delta log stats just like the CPU does when not using deletion vectors.  I could see shipping without full deletion support, but we should be adding these deletion metrics even when we don't support them being non-zero.  I'm OK with that being a followup, but IMO having the metric is a P0 for the release while the full support issue is not.

Reply:
This change should not have been in this PR. I have copied your comment into https://github.com/NVIDIA/spark-rapids/pull/8624#discussion_r1246899661 where we can discuss more. I have removed this change from this PR since this PR is all about falling back to CPU.